extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/templates/configmap.yaml

```diff
@@ -30,8 +32,17 @@ data:
       {{- end }}
       </property>
       {{- end }}
+    {{- else }}
+      <property>
+        <name>{{ $baseProperty.name }}</name>
+        <value>{{ $baseProperty.value }}</value>
+      {{- if $baseProperty.description }}
+        <description>{{ $baseProperty.description }}</description>
+      {{- end }}
+      </property>
```

Comment:
Q: Should this else be here. It seems like we're still adding the base property regardless of the outcome of `{{- if not (contains "true" $val) }}`. Would think if that if false, then we don't want to add the base property here.

Reply:
Hi! this else statement is for this check (only applicable if properties is empty.)

{{- if $.Values.configMap.metastoreServiceConfig.properties }}

 Looking at the logic seems like this isn't really needed as passing empty properties is safe. Ill clean it up!

Reply:
sounds good Ill keep as is!